pkg/controllers/disruption/orchestration/queue.go

```diff
@@ -345,11 +330,12 @@ func (q *Queue) Remove(cmd *Command) {
 	q.mu.Unlock()
 }
 
-// Reset is used for testing and clears all internal data structures
-func (q *Queue) Reset() {
+func (q *Queue) Reset(rateLimitingInterface workqueue.RateLimitingInterface) { // nolint:staticcheck
```

Comment:
This seems odd. Why do we need to have a parameter here? can't we just keep workqueue.New()?

Reply:
Agree that it seems odd but I tried round and round and came up with this as the best option. I tried re-using the existing queue, but clearing out the queue is tricky business due to the threads that add items into the queue occurring completely in parallel from the execution of the test.

Tried it a few different ways and couldn't get it to work, so the only way that I could get this to work was to fully replace the RateLimitingInterface.

Reply:
> clearing out the queue is tricky business due to the threads that add items into the queue occurring completely in parallel from the execution of the test

Why do you care about clearing out the queue? can't you just nuke it? are you worried about a memory leak?

Reply:
Same thing. Nuke it, clear it by using the same queue -- either works -- I'd prefer re-using the same queue which is why I was going that way. Again, seems not possible.